backend/models/notification.js

```diff
@@ -18,7 +18,7 @@
 "use strict";
 const { hasWriteAccessToModelHelper, hasReadAccessToModelHelper } = require("../middlewares/checkPermissions");
 const modelSettings = require("../models/modelSetting");
-const job = require("./job");
+const Job = require("./job");
```

Comment:
Can we change this to only import the functions needed in this file? this will reduce the chance of cyclic dependencies (I think)